specification/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Will it be better if put this file in specification/common-types/cadl?  as common-types is the place where cross RP sharing file, and for api spec pipeline, We already a job to sync common-types folder to ado repo.

Reply:
package.json file must live in a directory in the hierarchy from where the cadl spec lives otherwise folks cannot just call npm install from that directory to get the dependencies they need installed. For that reason we need the file to live in one of the parent folders to enable the basic install/compile commands to work.